app/user/tests/test_serializer.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from app.user.models import Profile
+
+User = get_user_model()
+
+
+class ImageUploadTest(TestCase):
+    def test_get_image(self):  # type: ignore
```

Comment:
no need for ignore everywhere it raises an errror otherwise what's the point of using mypy